seatunnel-dist/release-docs/LICENSE

```diff
@@ -329,7 +329,8 @@ The text of each license is the standard Apache 2.0 license.
      (Apache License, Version 2.0) Apache Hadoop YARN Client (org.apache.hadoop:hadoop-yarn-client:3.0.0 - no url defined)
      (Apache License, Version 2.0) Apache Hadoop YARN Common (org.apache.hadoop:hadoop-yarn-common:3.0.0 - no url defined)
      (Apache License, Version 2.0) Apache HttpAsyncClient (org.apache.httpcomponents:httpasyncclient:4.1.2 - http://hc.apache.org/httpcomponents-asyncclient)
-     (Apache License, Version 2.0) Apache HttpClient (org.apache.httpcomponents:httpclient:4.5.2 - http://hc.apache.org/httpcomponents-client)
+     (Apache License, Version 2.0) Apache HttpClient (org.apache.httpcomponents:httpclient:4.5.13 - http://hc.apache.org/httpcomponents-client)
+     (Apache License, Version 2.0) Apache Commons Codec (commons-codec:commons-codec:1.11 - https://commons.apache.org/proper/commons-codec)
```

Comment:
@zhaomin1423 You shouldn’t add this line, please delete this line, which is displayed at log as well. Cc @CalvinKirs

Reply:
I think this is a bug of Python check, this jar should be added. (if you look at his dependencies).

Reply:
@CalvinKirs Emm.. I'm afraid this is not about the Python script itself, the reason was that the `license-maven-plugin` maven plugin is not stable when the sortArtifactByName option is enabled, and I will raise up another PR to enhance the plugin.

Reply:
OK, I will merge this PR and look forward to your refinement.

Reply:
@CalvinKirs Agree, thanks

Reply:
It has been solved by https://github.com/apache/incubator-seatunnel/pull/1387